README.md

```diff
@@ -109,7 +109,7 @@ After the initial backup, use the following commands:
  - The `ghe-backup` command creates incremental snapshots of repository data,
    along with full snapshots of all other pertinent data stores.
  - The `ghe-restore` command restores snapshots to the same or separate GitHub
-   Enterprise appliance.
+   Enterprise appliance. You must add the backup host's SSH key to the target GitHub Enterprise appliance before using this command.
```

Comment:
It looks like the text is wrapping in the [rendered view](https://github.com/mellybess/backup-utils/blob/master/README.md#using-the-backup-and-restore-commands), is there something else I need to do?